### PR TITLE
Run optipng over some image resources

### PR DESCRIPTION
**Pull request by dbrgn**
I first ran [optipng](http://optipng.sourceforge.net/) (with `-o7`) over all images and then checked for
which ones the file size reduction was significant. Here's the diff
(before/after):

    36c36
    < 8.0K  img/actions/cam_export.png
    ---
    > 4.0K  img/actions/cam_export.png
    59c59
    < 44K   img/actions/select.png
    ---
    > 16K   img/actions/select.png
    65c65
    < 8.0K  img/actions/save.png
    ---
    > 4.0K  img/actions/save.png
    115c115
    < 12K   img/app/librepcb.png
    ---
    > 8.0K  img/app/librepcb.png

In summary, this commit losslessly reduces the file size of four of the
PNG icons. File size reduction ranges from 33% to 64%.

### CODE REVIEW

**Comment by dbrgn** [comment]
It was no issue, but it's reduced file size for free, with no loss of quality :)

**Comment by ubruhin** [comment]
> I guess it will dramatically reduce the loading time of LibrePCB, since it has less data to load from the disk

Sure, but then cloning this repository will *dramatically* require more time because of the larger history :wink:

**Comment by ubruhin** [comment]
@dbrgn How did you compare the file sizes? It seems that your results are rounded to 4KB blocks. When comparing the size in bytes, it looks different:

master:
```
du -b actions/cam_export.png actions/save.png actions/select.png app/librepcb.png 
4927	actions/cam_export.png
4150	actions/save.png
42855	actions/select.png
8940	app/librepcb.png
```

optipng:
```
du -b actions/cam_export.png actions/save.png actions/select.png app/librepcb.png 
4036	actions/cam_export.png
4057	actions/save.png
13618	actions/select.png
6593	app/librepcb.png
```

**Comment by dbrgn** [comment]
I either used "du -h" or "ls -lah", can't remember.

Anyways, don't spend too much time with this. If you don't think it's worth it, just close the issue.

**Comment by ubruhin** [comment]
OK I close this because the file size reduction is marginal while file size isn't an issue at all. Sorry...
